Reference_Manual.md

```diff
@@ -2060,7 +2065,7 @@ The `lifecycle_listener` interface notifies for the following events:
 
 The following is an example of the `lifecycle_listener` that is added to the `client_config` object and its output.
 
-```C++
+```c++
     class ConnectedListener :
```

Comment:
```suggestion
```

Reply:
removed this line and the following lines.